platform/android/export/export_plugin.cpp

```diff
@@ -69,6 +69,7 @@ static const char *android_perms[] = {
 	"ACCESS_FINE_LOCATION",
 	"ACCESS_LOCATION_EXTRA_COMMANDS",
 	"ACCESS_MOCK_LOCATION",
+	"ACCESS_MEDIA_LOCATION",
```

Comment:
The entry should come before `access_mock_location`

Reply:
forgot the alphabet 😅